modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupAnalyticsCloudForm/index.js

```diff
@@ -9,14 +9,66 @@
  * distribution rights of the Software.
  */
 
+import {useQuery} from '@apollo/client';
 import ClayForm from '@clayui/form';
-import {Formik} from 'formik';
-import {isLowercaseAndNumbers} from '../../../../common/utils/validations.form';
+import {FieldArray, Formik} from 'formik';
+import {useEffect, useMemo} from 'react';
+import {getDXPCloudPageInfo} from '../../../../common/services/liferay/graphql/queries';
+import {
+	hasFriendlyURL,
+	hasNoEmptySpace,
+	isLowercaseAndNumbers,
+	isStartingWithSlash,
+	maxLength,
+} from '../../../../common/utils/validations.form';
 import {Button, Input, Select} from '../../../components';
-
+import getInitialAnalyticsInvite from '../../../utils/getInitialAnalyticsInvite';
 import Layout from '../Layout';
+import IncidentReportInput from './IncidentReportInput';
+
+const INITIAL_SETUP_ADMIN_COUNT = 1;
+
+const SetupAnalyticsCloudPage = ({
+	handlePage,
+	leftButton,
+	project,
+	setFieldValue,
+	values,
+}) => {
+	const {data} = useQuery(getDXPCloudPageInfo, {
+		variables: {
+			accountSubscriptionsFilter: `(accountKey eq '${project?.dXPCDataCenterRegions}') and (hasDisasterDataCenterRegion eq true)`,
+		},
+	});
```

Comment:
use client.query here instead. Call directly on the useEffect hook

Reply:
ok, thank you!